### ISSUE TITLE
Citation collapsing either not implemented or not working

### ISSUE THREAD

**Issue by krafczyk** [inukshuk/jekyll-scholar#196]
Certain citation styles enable citation collapsing.
In numeric citation styles this means [1],[2],[3] -> [1-3] and [1],[2],[3],[5],[6],[8]->[1-3,5,6,8]
See: https://github.com/citation-style-language/documentation/blob/master/specification.txt#L1655

ieee is a numberic citation style, and using this with my jekyll website results in non-collapsed citations. I suspect that either citation collapsing is either not implemented or not working.

To test this, I've made a test cite which you clone here: https://github.com/krafczyk/citetest-page

clone, then run `bundle exec jekyll serve` and navagate to `localhost:4000` to see what I mean.

This is how my machine renders the page:
![citetest-page-render](https://user-images.githubusercontent.com/6449564/28290556-76eb7ac2-6b0c-11e7-8416-9f1651c53d77.png)

the [1],[2],[3],[4],[5] should instead be [1-5].

**Comment by inukshuk** [comment]
Yes, you're right, cite-collapsing is not implemented in [citeproc-ruby](https://github.com/inukshuk/citeproc-ruby); we'll have to add it there for it to work in jekyll scholar.
